docs/user/nifi.rst

```diff
@@ -0,0 +1,179 @@
+GeoMesa NiFi Plugins
+====================
+
+NiFi allows for managing large batches and streams of files and data.
+GeoMesa-NiFi allows you to ingest data into GeoMesa strait from NiFi by
+leveraging custom processors.
+
+Installation
+------------
+
+Install the Processors
+^^^^^^^^^^^^^^^^^^^^^^
+
+Pick a reasonable directory on your machine, and run:
+
+.. code-block:: bash
+
+    $ git clone https://github.com/geomesa/geomesa-nifi.git
+    $ cd geomesa-nifi
+
+To build, run
+
+.. code-block:: bash
+
+    $ mvn clean install
+
+
+To install the GoeMesa processors you will need to copy the geomesa-nifi-nar file from
+``geomesa-nifi/geomesa-nifi-nar/target/geomesa-nifi-nar-$VERSION.nar`` into the ``lib/`` directory of your NiFi installation.
+
+Install the SFTs and Converters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The GeoMesa processors need access to the SFTs and Converters in order to ingest data. We can provide these to the
```

Comment:
Also, can we put a reference to the converters chapter, which talks more about the format of the reference.conf file?